### PR TITLE
1.22 Manual: add graphql docs

### PR DESCRIPTION
**Pull request by timogoebel**

### CODE REVIEW

File: _includes/manuals/1.22/5.1_api.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-[API v2](api/{{page.version}}/index.html) is the default, stable and recommended version for Foreman {{page.version}}.  API v1 is also available, but future versions of Foreman will eventually deprecate and remove it.
```
Comment (ekohl) [inline _includes/manuals/1.22/5.1_api.md:1]: IIRC we removed v1 a while back (1.20? 1.21?) so we should make sure to update those manuals as well.

**Review by ekohl — COMMENTED** [review]
Thanks! It looks good. Could you also apply this to the nightly docs?

File: _includes/manuals/1.22/5.1_api.md (hunk 1 above)
Comment (timogoebel) [reply _includes/manuals/1.22/5.1_api.md:1]: Actually, it was 1.18, see https://projects.theforeman.org/issues/11389.

File: _includes/manuals/1.22/1.2_release_notes.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ As a part of this change searching by parameter value is no longer allowed. ([#4
 
 Google compute resource has been fixed now. It was previously not functional due to changes in Google compute APIs. `cloud-init` integration has also been added to Google compute resource. There are still a few edge cases which need to be handled, but basic provisioning should be functional.
 
+#### Graphql API
+The 1.22 release of Foreman now contains a new, experimental, API based on [GraphQL](https://graphql.org/). Graphql allows clients to ask for specific data which can reduce round trips. Future releases will add more queries to the API.
```
Comment (tbrisker) [inline _includes/manuals/1.22/1.2_release_notes.md]: Perhaps add a link to the graphql section as well for further details?

**Comment by timogoebel** [comment]
Updated.

**Review by tbrisker — APPROVED** [review]
Thanks @timogoebel !

**Pull request merged**